### PR TITLE
#19240 Fix ECR Vulnerabilities: thecodingmachine/gotenberg (Deadline: November 06, 2023)

### PR DESCRIPTION
**Pull request by nangirvan**
- [x] Upgrade `thecodingmachine/gotenberg` from `7.8.3` to [7.9.2](https://hub.docker.com/r/thecodingmachine/gotenberg/tags) ([changelog](https://github.com/gotenberg/gotenberg/releases/tag/v7.9.2))

### CODE REVIEW

**Comment by nangirvan** [comment]
@daniel-jeremia please help to review ya, thankyouu

File: charts/datasaur/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.8.1
+version: 1.9.0
```
Comment (daniel-jeremia) [inline charts/datasaur/Chart.yaml]: this should be a patch update not a minor update @nangirvan

Comment (nangirvan) [reply charts/datasaur/Chart.yaml]: @daniel-jeremia updated

File: charts/datasaur/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.8.1
+version: 1.8.2
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
```
Comment (daniel-jeremia) [inline charts/datasaur/Chart.yaml:21]: tolong update appVersion nya juga ya, jadi `>=6.37.0` @nangirvan

**Comment by nangirvan** [comment]
based on this thread https://github.com/vulcan-ai/vulcan/pull/19507#discussion_r1375716757, we decided to not upgrade it because there are many vulnerabilities in the newest version

**Pull request closed without merging**